src/database/standalone-sqlite.ts

```
@@ -3450,9 +3450,9 @@ if (!isMainThread) {
           worker.saveNestedDataHash(args[0]);
           parentPort?.postMessage(null);
           break;
-        case 'getUnverifiedDataIds':
-          const unverifiedIds = worker.getUnverifiedDataIds();
-          parentPort?.postMessage(unverifiedIds);
+        case 'getVerifiableDataIds':
+          const ids = worker.getVerifiableDataIds();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)